apps/src/publicKeyCryptography/CollapsiblePanel.jsx

```diff
@@ -0,0 +1,35 @@
+/** @file Collapsible panel with title, used for each character in crypto widget */
```

Comment:
Hah, didn't read PR caveats closely enough and was like... where's the collapsing! TODO could be nice but doesn't matter if still actively developing solo :).